pluginframework/common/converters_test.go

```diff
@@ -0,0 +1,256 @@
+package pluginframework
```

Comment:
Can we move these files into `internal/pluginframework/converters`? Keep functions internal that don't need to be exposed publicly, plus converters is a better package name than common or pluginframework.